theClanless/src/main/resources/theClanlessResources/localization/eng/DefaultMod-Card-Strings.json

```diff
@@ -53,16 +89,31 @@
     "DESCRIPTION": "Gain !M! theclanless:maneuvers and draw a card.",
     "UPGRADE_DESCRIPTION": "Gain !M! theclanless:maneuvers and draw a card."
   },
+  "theClanless:ForcedVigilance": {
+    "NAME": "Forced Vigilance",
+    "DESCRIPTION": "Gain 1 energized for up to !M! unused energy at the end of your turn.",
```

Comment:
Doesn't this, er, double dip with ice cream cone?

Reply:
yes, but it's limited to the STACK of the power, so yes, if you get a couple of these and ice cream cone you will have a good game. I've done WORSE with the Watcher.